src/popup/components/AuctionOverview.vue

```diff
@@ -2,58 +2,65 @@
   <div class="auction-overview">
     <DetailsItem :label="$t('pages.auctionBid.current-highest-bid')">
       <template #value>
-        <TokenAmount
-          :amount="+getHighestBid(name).nameFee"
-        />
+        <TokenAmount :amount="amount" />
       </template>
     </DetailsItem>
     <DetailsItem
       class="end-height"
       :label="$t('pages.auctionBid.ending-height')"
       :value="auction.expiration"
-      :secondary="`(≈${blocksToRelativeTime(blocksToExpiry)})`"
+      :secondary="`(≈${endHeight})`"
     />
   </div>
 </template>
 
-<script>
-import { pick } from 'lodash-es';
-import { mapGetters } from 'vuex';
-import { blocksToRelativeTime } from '../utils';
+<script lang="ts">
+import { computed, defineComponent } from '@vue/composition-api';
+import { IAuction, IAuctionBid } from '../../types';
+import { useGetter } from '../../composables';
+import { blocksToRelativeTime, rxJsObservableToVueState } from '../utils';
+
 import DetailsItem from './DetailsItem.vue';
 import TokenAmount from './TokenAmount.vue';
 
-export default {
+export default defineComponent({
   components: {
     DetailsItem,
     TokenAmount,
   },
   props: {
     name: { type: String, required: true },
   },
-  subscriptions() {
-    return pick(this.$store.state.observables, ['topBlockHeight']);
-  },
-  computed: {
-    ...mapGetters('names', ['getAuction', 'getHighestBid']),
-    auction() {
-      return this.getAuction(this.name);
-    },
-    blocksToExpiry() {
-      return this.auction.expiration - this.topBlockHeight;
-    },
-  },
-  methods: {
-    blocksToRelativeTime,
+  setup(props, { root }) {
+    const topBlockHeight = rxJsObservableToVueState<number>(
+      (root.$store.state as any).observables.topBlockHeight,
+    );
+
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Are we able to remove that comment and resolve the problems related to it?

Reply:
Unfortunately not in our TS setup. I tried, but this again requires raising TS version :/ With Vue 3 this would work nice.